javascript/js-declarations/src/org/jetbrains/dukat/js/declarations/JSModuleDeclaration.kt

```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.dukat.js.declarations
+
+import org.jetbrains.dukat.js.declarations.toplevel.JSDeclaration
+
+data class JSModuleDeclaration(
+        val moduleName: String,
+        val fileName: String,
+        val exportDeclarations: MutableSet<JSDeclaration>,
```

Comment:
I've restricted the exports to only use a single value now, since the exports variable in JS can only hold a single value ether, and will have to hold an object, if multiple exports are required (I'll be working on that part in the future)

This Set was a remnant from a previous design, where something else was stored here.